src/PolyhedralGeometry/Polyhedron/properties.jl

```diff
@@ -705,6 +705,48 @@ julia> normalized_volume(C)
 normalized_volume(P::Polyhedron) =
   coefficient_field(P)(factorial(dim(P)) * (pm_object(P)).VOLUME)
 
+@doc raw"""
+    castelnuovo_excess(P::Polyhedron)
+
+For an arbitrary lattice polytope, Hibi [Hib94](@cite) proved that the normalized volume is always at least as large as a certain lattice point count.
+This function returns the difference between those two numbers.
+
+# Examples
+```jldoctest
+julia> castelnuovo_excess(cube(4))
+154
+```
+"""
+function castelnuovo_excess(P::Polyhedron)::Integer
```

Comment:
```suggestion
function castelnuovo_excess(P::Polyhedron)
```
We usually don't annotate return values as that sometimes confuses type inference and leads to a bunch of invalidations